src/PriceIndicator/PriceIndicator.styles.ts

```diff
@@ -51,14 +55,18 @@ export const Divider = styled(motion.div)`
   left: 0;
 `;
 
-export const TotalPrice = styled.div<{ isActive: boolean }>`
+export const TotalPrice = styled.div<{ isActive: boolean; isPriceHover: boolean }>`
   display: inline-flex;
   position: relative;
   justify-content: center;
   align-items: center;
   height: 32px;
+  margin-top: ${p => p.theme.spacing.small};
+  margin-bottom: ${p => p.theme.spacing.small};
   padding: 0 ${p => p.theme.spacing.xsmall};
   border-radius: ${p => p.theme.borderRadius.standard};
+  border-top-left-radius: ${p => (p.isPriceHover ? '0px' : p.theme.borderRadius.standard)};
+  border-bottom-left-radius: ${p => (p.isPriceHover ? '0px' : p.theme.borderRadius.standard)};
```

Comment:
Are you able to just use isPriceHover without the p. here? since it is destructured above?

Reply:
Nah, the destructuring is just the types of the props to be passed in, we still need to get them from the `p` in the arrow function.